app/DoctrineMigrations/Version20150902154411.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Surfnet\StepupGateway\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20150902154411 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE TABLE registration (key_handle VARCHAR(255) NOT NULL, public_key VARCHAR(255) NOT NULL, sign_counter INT NOT NULL, PRIMARY KEY(key_handle)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
```

Comment:
For all: please add schema-name, prevents this to be ran on the wrong schema and at the same time guarantees that the executor has correct rights. Keep in mind that the schema is configurable. 

Also, not knowing the spec, is 255 chars enough for a public key? Might want to up that a little. The keyhandle seems to be a sha256, so that might actually be a bit smaller? Not sure if the latter is worth the hassle, but the former might be an issue...